src/content/tools/dartpad/index.md

```diff
@@ -5,7 +5,7 @@ description: The tool that lets you interactively play with Dart in a browser.
 
 DartPad is an [open source tool]({{site.repo.dart.org}}/dart-pad)
 that lets you play with the Dart language in any modern browser.
-Many pages in this site—especially [codelabs](/codelabs)—have
+Many pages in this site—especially [codelabs](/tutorials)—have
```

Comment:
**issue:** Same as [line 2778](https://github.com/dart-lang/site-www/pull/5858/files#diff-75ed5ce4cf7ea66ddd2f5aa89546afe28498b1afb33bcf6a6f56fb7407445cd6R2778) in [js-to-dart.md](src/content/resources/coming-from/js-to-dart.md)

```suggestion
Many pages in this site, like the [tutorials](/tutorials), have
```